active/0002-centalized-mgmt-resource-types-provider-versions.md

```diff
@@ -107,12 +111,12 @@ Bicep will replace-merge the contents in the `bicepconfig.json` with the default
 `bicepconfig.json`
 ```json
 {
- "providerImports": {
-    "az": "mcr.microsoft.com/bicep/providers/az:0.2.3",
-    "kubernetes": "builIn",
-    "microsoftGraph": "builtIn",
+ "providers": {
+    "az": "br:mcr.microsoft.com/bicep/providers/az:0.2.3",
+    "kubernetes": "builin:",
```

Comment:
I assume it's a typo here and should be `builtin:`?

Reply:
yep, it as a typo.